lua/nordic/init.lua

```diff
@@ -31,6 +29,6 @@ end, {
     nargs = 1,
 })
 
-M.setup = require 'nordic.config'.setup
+M.setup = require('nordic.config').setup
```

Comment:
The formatter (ran when merged into main) takes care of these things.

Reply:
Thats cool